.eslintignore

```diff
@@ -3,6 +3,8 @@ lint-staged.config.js
 scripts/serve.js
 
 .yarn-packages
+.yarn
```

Comment:
can we use the config to _not_ use junk up the root with `pnp`? Or not use the `pnp` features at all?

Reply:
isn't `.pnp` already ignored on the line below?

Reply:
The point would be to remove all the added `.yarn` and `.pnp` entries  from the `.gitignores` _and_ never see them on disk
- are we even using `pnp`? 
  - let's not create `.pnp`/`.yarn` at all, if possible
  - we're using `node_modules`
- if we _do_ need to create it for `nx` or `lerna` or whatever to be happy, 
  - can we put it in `build/.(pnp|yarn)` or something

As I've been exploring these new "features," (still haven't found what value the new `yarn` adds, other than an incompatible lockfile) I've ended up with some patterns like this to improve the cacheability of `node_modules` and reduce the extreme amount of noise from e.g. `jlpm`:

```yaml
enableInlineBuilds: false
enableTelemetry: false
httpTimeout: 60000
nodeLinker: node-modules
npmRegistryServer: https://registry.npmjs.org/
installStatePath: ./build/.cache/yarn/install-state.gz
cacheFolder: ./build/.cache/yarn/cache
logFilters:
  - code: YN0006
    level: discard
  - code: YN0002
    level: discard
  - code: YN0007
    level: discard
  - code: YN0013
    level: discard
  - code: YN0019
    level: discard
  - code: YN0008
    level: discard
```

As well as some env vars:

```py
      "NX_CACHE_DIRECTORY": f"{P.ROOT}/build/.cache/nx",
      "NX_PROJECT_GRAPH_CACHE_DIRECTORY": f"{P.ROOT}/build/.cache/nx",
      "YARN_CACHE_FOLDER": f"${P.ROOT}/build/.cache/yarn"
```

Reply:
Any reason we should have this configuration here in the JupyterLite repo and not upstream in the JupyterLab repo?

If not, should we open an issue about it in the JupyterLab repo? (or maybe there is already one)

Reply:
oh, and this could _also_ go into `package.json` (unlike `prettierignore`, for some reason, buh)